packages/karma/karma.conf.js

```diff
@@ -188,8 +188,12 @@ try {
    */
   function configureFiles(conf) {
     overrideConfigValue(conf, 'files', [
+      // BEGIN BOOTSTRAP FILES
```

Comment:
I like the idea of annotating the karma.conf to explain where things came from - are there more inputs we could associate here with where they ended up?

Reply:
yes, all templated inputs are now annotated.